vmdb/app/views/layouts/listnav/_container_node.html.haml

```diff
@@ -12,6 +12,16 @@
         %li
           = link_to(_('Summary'), {:action => 'show', :id => @record, :display => 'main'}, :title => _("Show Summary"))
 
+        - if @record.has_events? || @record.has_events?(:policy_events)
```

Comment:
@simon3z Is it possible for the left side to be false but the right side true?  It seems like has_events? is badly named if so.